mix.exs

```diff
@@ -26,7 +26,7 @@ defmodule BroadwayRabbitMQ.MixProject do
 
   defp deps do
     [
-      {:broadway, "~> 0.2.0"},
+      {:broadway, github: "plataformatec/broadway"},
```

Comment:
Broadway v0.3.0 has just been released ;)

Reply:
Can I safely rebase from master then ?

Reply:
Will update the dependency as well !